exceptions.h

```diff
@@ -0,0 +1,25 @@
+/*****************************************************************************\
```

Comment:
I'm not sure we need this custom exception class, because we never throw it, do we?

Reply:
We catch this in common_clang.cpp:346 as std::domain_error, do you mean this kind of exception will never be hit?